run/pubsub/main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 // PubSubMessage is the payload of a Pub/Sub event.
 // See the documentation for more details:
 // https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage
+// byte slice unmarshalling handles base64 decoding
```

Comment:
I understand why the comment is here but should it be on line 71?

Reply:
maybe? I contemplated that, but I didn't want to break up the code either.